pinot-query-planner/src/main/java/org/apache/pinot/query/QueryEnvironment.java

```diff
@@ -172,12 +173,12 @@ public String explainQuery(String sqlQuery, SqlNodeAndOptions sqlNodeAndOptions)
 
   @VisibleForTesting
   public QueryPlan planQuery(String sqlQuery) {
```

Comment:
This function is only called from tests, right? 

The actual code path is `MultiStageBrokerRequestHandler.handleRequest()` -> `QueryEnvironment.planQuery(sqlQuery, sqlNodeAndOptions)`.

Reply:
ahh good catch. yes i think in this case we need multiple entrypoint to support isRemoveComment flag. will follow up